multimodal/src/autogluon/multimodal/utils/environment.py

```diff
@@ -335,11 +335,11 @@ def get_available_devices(num_gpus: int, auto_select_gpus: bool):
 
 def _get_mmlab_installation_guide(package_name):
     if package_name == "mmdet":
-        err_msg = 'Please install MMDetection by: pip install "mmdet>=3.0.0".'
+        err_msg = 'Please install MMDetection by: pip install "mmdet==3.2.0".'
     elif package_name == "mmcv":
-        err_msg = "Please install MMCV by: mim install mmcv"
+        err_msg = 'Please install MMCV by: mim install "mmcv==2.1.0"'
     elif package_name == "mmengine":
-        err_msg = "Please install MMCV by: mim install mmengine"
+        err_msg = 'Please install MMCV by: mim install "mmengine<0.8"'
```

Comment:
This message should say installing mmcv?

Reply:
Thanks! Should be "Please install MMEngine". Fixed.